### PR TITLE
 docs - update mlbparks example

### PR DESCRIPTION
**Pull request by kadel**

### CODE REVIEW

File: docs/example.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
 # Full workflow tutorial
 
-This example demonstrates how to export application deployed on Openshift,
-to Nulecule specification and then import that application to Kubernetes.
+This example demonstrates how to export application deployed on Openshift
+to Nulecule specification and then import that application to Kubernetes and/or OpenShift.
 
 This is a 3 step tutorial:
```
Comment (surajssd) [inline docs/example.md]: Also adding a blank line after this will make the 1. 2. and 3. get onto separate lines.

File: docs/example.md [hunk 2]
```diff
@@ -103,13 +104,18 @@ This has been tested on ADB 1.7.1 as Fedora 23 on local machine
...
 
  1. Add Docker Registry on you local machine as insecure
-     
+    Add your local machine's Docker registry (which was started in the first step).
     - `sudo vi /etc/sysconfig/docker`
     - Change line `# INSECURE_REGISTRY='--insecure-registry '` to `INSECURE_REGISTRY='--insecure-registry 10.1.2.1:5000'`
     - `sudo systemctl restart docker`
 
+
+ 1. Label /var/lib/kubelet/pods as `svirt_sandbox_file_t`
```
Comment (surajssd) [inline docs/example.md]: Back ticks for /var/lib/kubelet/pods.

File: docs/example.md [hunk 3]
```diff
@@ -127,7 +133,12 @@ This has been tested on ADB 1.7.1 as Fedora 23 on local machine
 
  1. Run mlbparks Nulecule app on OpenShift
     - `oc new-project mlbimport`
-    - `atomic run 10.1.2.1:5000/mlbparks-atomicapp --provider=openshift --providerconfig=/home/vagrant/.kube/config --namespace=mlbimport`
+    - `sudo -E atomic run 10.1.2.1:5000/mlbparks-atomicapp --provider=openshift --providerconfig=/home/vagrant/.kube/config --namespace=mlbimport`
+ 1. Wait for mlbparks build to finish
+    You can see log of build on web console, or in you terminal using this command `oc build-logs mlbparks-1`.
+
+    Because of OpenShift Origin issue [#4518](https://github.com/openshift/origin/issues/4518) and Atomic App issue [#669](https://github.com/projectatomic/atomicapp/issues/669) build might fail on `Error: build error: Failed to push image. ....... : no basic auth credentials`. Just starting build again with `oc start-build mlbparks` should be enought to fix this.
```
Comment (surajssd) [inline docs/example.md]: Typo: s/enought/enough

File: docs/example.md [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
-This example demonstrates how to export application deployed on Openshift,
-to Nulecule specification and then import that application to Kubernetes.
+This example demonstrates how to export application deployed on Openshift
+to Nulecule specification and then import that application to Kubernetes and/or OpenShift.
 
 This is a 3 step tutorial:
     1. Deploy a sample application on Openshift.
     2. Export the application using openshift2nulecule.
     3. Import the exported artifacts into Kubernetes using Atomic App.
```
Comment (surajssd) [inline docs/example.md]: Import the exported artifacts into Kubernetes and/or OpenShift using Atomic App.

File: docs/example.md [hunk 5]
```diff
@@ -51,47 +54,45 @@ This has been tested on ADB 1.7.1 as Fedora 23 on local machine
...
-        enabled=1
-        EOF
-        "
-        ```
+
+    This step is **not** required if you are using CDK 2.0. OpenShift2Nulecule should be already installed in CDK box.
+
+    - Add [tkral/openshift2nulecle](https://copr.fedorainfracloud.org/coprs/tkral/openshift2nulecle/) Copr repository
+      
+      `sudo sh -c "curl  https://copr.fedorainfracloud.org/coprs/tkral/openshift2nulecle/repo/epel-7/tkral-openshift2nulecle-epel-7.repo > /etc/yum.repos.d/tkral-openshift2nulecle-epel-7.repo"`
```
Comment (surajssd) [inline docs/example.md]: I feel we should add the install information in https://github.com/projectatomic/openshift2nulecule/blob/master/docs/installation.md#usage-installation 

Here you can divide the 'Usage installation' into two as, "Install as RPM" and "Install from Source". So the above steps add under "Install as RPM" and here just put a link to that topic.

**Comment by kadel** [comment]
I'm going to merge this, because we going to work on this with Bex

File: docs/example.md [hunk 6]
```diff
@@ -103,13 +104,18 @@ This has been tested on ADB 1.7.1 as Fedora 23 on local machine
...
-     
+    Add your local machine's Docker registry (which was started in the first step).
     - `sudo vi /etc/sysconfig/docker`
     - Change line `# INSECURE_REGISTRY='--insecure-registry '` to `INSECURE_REGISTRY='--insecure-registry 10.1.2.1:5000'`
     - `sudo systemctl restart docker`
 
+
+ 1. Label /var/lib/kubelet/pods as `svirt_sandbox_file_t`
+    This step is because of bug [#117](https://github.com/projectatomic/adb-atomic-developer-bundle/issues/117) in ADB 
+    - `chcon -Rt svirt_sandbox_file_t /var/lib/kubelet/pods`
```
Comment (surajssd) [inline docs/example.md]: Needs `sudo` before `chcon`.

**Comment by surajssd** [comment]
@kadel np, I also tried all the steps and it works fine!

**Comment by kadel** [comment]
> @kadel np, I also tried all the steps and it works fine!

Great!  :tada:  Thans a lot for testing it.

**Pull request merged**